resources/assets/js/components/JobPreview.tsx

```diff
@@ -68,67 +150,183 @@ const JobPreview: React.FunctionComponent<JobPreviewProps> = ({
         </i>
         {city}, {province}
       </p>
-      {remoteWork && (
-        <p>
-          <i
-            data-c-colour="c1"
-            className="fas fa-home"
-            title="Remote Work Icon."
-          >
-            &nbsp;&nbsp;
-          </i>
-          Remote Work Allowed
-        </p>
-      )}
+      <p>
+        <i data-c-colour="c1" className="fas fa-home" title="Remote Work Icon.">
+          &nbsp;&nbsp;
+        </i>
+        {remoteWork
+          ? intl.formatMessage(messages.allowed)
+          : intl.formatMessage(messages.notAllowed)}
+      </p>
       <h4
         data-c-font-size="h4"
-        data-c-font-weight="bold"
+        data-c-font-weight="600"
         data-c-margin="top(double) bottom(normal)"
+        data-c-border="bottom(thin, solid, black)"
+        data-c-padding="bottom(normal)"
       >
-        Basic Information
+        <FormattedMessage
+          id="jobPreview.basicInformation"
+          defaultMessage="Basic Information"
+          description="Section Header for basic information"
+        />
       </h4>
       <div data-c-grid="gutter">
         <div data-c-grid-item="tp(1of2)">
           <p data-c-colour="c1" data-c-margin="bottom(quarter)">
-            Average Annual Salary
+            <FormattedMessage
+              id="jobPreview.lengthOfTheTerm"
+              defaultMessage="Length of the Term"
+              description="Datum Label"
```

Comment:
I know it's kind of annoying to have to fill out all of these descriptions, but I believe the spirit of the property is to add context to the line being translated. When a translator receives a file with hundreds of items needing translation, it's not very clear what the context is surrounding the lines.

Having `Datum Label` in this case for most of the translations in this file isn't super helpful to that end.

Reply:
Agreed I would like something that provides context to the header. how is? "Job Preview Card Info Label"